dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/JSONFieldsTest.java

```diff
@@ -166,14 +167,14 @@ public void testMessageFieldNamesOmitXML() throws Exception {
 
         setServerConfiguration(true, "message:", server_xml);
         server_xml.setMarkToEndOfLog();
-        assertNull("The message field name was not omitted in messages.log.", server_xml.waitForStringInLogUsingMark("\"message\""));
+        assertNull("The message field name was not omitted in messages.log.", server_xml.waitForStringInLogUsingMark("\"message\"", LOG_WAIT_TIMEOUT));
```

Comment:
Not sure what is the purpose of this test.  
1. It changed the server configuration.  
2. Set log mark to the end
3. Wait for "message"
There was no operation between step 2 and 3 and nothing would happen and the wait operation will wait until timeout 100%.